pkgs/by-name/im/immersed/package.nix

```diff
@@ -11,12 +11,12 @@ let
 
```

Comment:
You forgot to update the version itself

Reply:
d'oh! ><

attempted to fix it just now. Thank you